core/domain/state_domain_test.py

```diff
@@ -1457,13 +1457,7 @@ def test_get_content_id_mapping_needing_translations_with_interaction_translatio
                 'default_outcome': {},
                 'solution': {},
                 'feedback_1': {},
-                'ca_placeholder_0': {
-                    'hi': {
-                        'data_format': 'unicode',
-                        'translation': 'Placeholder translation',
-                        'needs_update': False
-                    }
-                },
+                'ca_placeholder_0': {},
```

Comment:
Are the above changes not tested anywhere? Can you add one ?

Reply:
This test was modified to expect 'ca_placeholder_0', which refers to an interaction.